docs/flashbots-protect/large-transactions.md

```diff
@@ -0,0 +1,7 @@
+---
+title: large transaction allowlist
```

Comment:
suggestion:

there's some discrepancy in capitalization, but this is the most common.

```suggestion
title: Large transaction allowlist
```